netdata-installer.sh

```diff
@@ -145,14 +145,14 @@ CFLAGS="${CFLAGS--O2}"
 ACLK="${ACLK}"
 
 # keep a log of this command
-# shellcheck disable=SC2129
-printf "\n# " >> netdata-installer.log
-date >> netdata-installer.log
-printf 'CFLAGS="%s" ' "${CFLAGS}" >> netdata-installer.log
-printf 'LDFLAGS="%s" ' "${LDFLAGS}" >> netdata-installer.log
-printf "%s" "${PROGRAM}" "${@}" >> netdata-installer.log
-printf "\n" >> netdata-installer.log
-
+{
+printf "\n# "
+date
+printf 'CFLAGS="%s" ' "${CFLAGS}"
+printf 'LDFLAGS="%s" ' "${LDFLAGS}"
+printf "%s" "${PROGRAM}" "${@}"
+printf "\n"
```

Comment:
Please indent things here so that it’s easier to visually track what is inside the braces.